RegTesting.Mvc/Views/Testcase/Index.cshtml

```diff
@@ -14,9 +14,6 @@
 		<th>
 			Activated
 		</th>
-		<th>
-			Type
```

Comment:
I think, the heading 'Type' should stay and the heading for 'Activated' should be deleted ;-)